src/cli/bin.js

```diff
@@ -2,10 +2,19 @@
 
 'use strict'
 
-const yargs = require('yargs')
+const pipe = require('pipe-args')
 const updateNotifier = require('update-notifier')
 const readPkgUp = require('read-pkg-up')
 
+const enableStdin = [
+  'files', 'path', 'object data', 'ref', 'domain-name', 'key', 'ipfs-path',
+  'name', 'address', 'data', 'peer', 'recursive', 'default-config', 'peer ID'
+]
```

Comment:
https://github.com/ipfs/go-ipfs/blob/2cc5ce45b875e45fdbfc779e1a7c4bef99bb9877/core/commands/dns.go#L46

It should be renamed to 'dns'